### DISCUSSION TITLE
Proposal for Increasing PoET Server Diversity to Promote Spacemesh Decentralization

### DISCUSSION THREAD

**Discussion by bugxzhu** [spacemeshos/go-spacemesh#5795] [General]
Dear Spacemesh Team,
I am a dedicated supporter of decentralized technologies and a keen user of the Spacemesh platform. I greatly admire your efforts and innovations in the field of decentralized computing.

However, I'd like to share some observations and suggestions regarding the current state of affairs. Particularly, I've noticed that the majority of miners tend to utilize H9 services. At the same time, there is a relative scarcity of publicly available PoET servers, which is not conducive to the development of small-scale miners.

Based on these observations, I'd like to propose a suggestion aimed at promoting diversity among PoET servers while encouraging more miners to participate in the Spacemesh ecosystem. Would it be possible to consider establishing a mechanism to incentivize PoET servers to provide public services through rewards? This approach could increase the number of PoET servers and encourage participation from different regions and communities, thereby better realizing Spacemesh's vision of decentralization.

I believe that by enhancing the diversity and quantity of PoET servers, we can improve the robustness and security of the Spacemesh network while also attracting more users and developers to join this ecosystem. I look forward to hearing your thoughts on this proposal and hope that we can work together to drive the growth and advancement of the Spacemesh platform.

Thank you for your attention and response!

**Comment by pigmej** [comment]
Hey,

POET incentives were always in sight. There are a few problems with doing that properly in the protocol, like who gets paid and how much, whether you pay only for the used fastest poet, whether you pay for all where you registered, etc.  More info about it is here https://github.com/spacemeshos/pm/issues/298

It's already being implemented. The poet will soon be able to obtain short-lived certificates via https://github.com/spacemeshos/pm/issues/290. That could be used to implement a custom off-chain paid poet service based on publicly available and in-node support for the certifier.

However, paid poet services are available on the network even now, mainly with custom implementation/logic.


Also, as a summary, when we will have VM, such stuff could be done based on a smart contract basis.

**Reply by noamnelke** [reply]
@bugxzhu thanks for the comment! As NJ wrote, this is something we're actively working on. We already have some community-run PoET servers and we're working to make providing such a service easy and profitable. Our high-level plan is to start by keeping it entirely off-protocol, but integrated into the node, so that paying for a faster PoET, or one that runs in a different phase is easy for non-technical smeshers (and for those providing the service!). This will be achieved through a PoET certification service, such that the node will obtain a (free/paid) certificate ahead of time and the PoET will verify the validity of the certificate as a precondition for registration.

This will allow PoET services to implement only a certification service and use the PoET as-is. At first, paid certification will have to be done manually, but as we discover how PoET services work, we'll make that process hands-free as well (by pre-signing transactions to the PoET and having the node transmit them at the right time).

Eventually, when we have our full VM, the plan is to support more elaborate payment schemes, like payment conditional on the PoET publishing a proof in time. We hope to learn from the forming ecosystem and adapt accordingly, rather than create a full "perfect" solution in advance.

**Reply by bugxzhu** [reply]
Thank you for your enthusiastic response and for sharing the exciting news.
@pigmej @noamnelke 

Another things。
There's a small yet growing rumor circulating within our community regarding the spread of a quick plot tool. This tool, as it appears, is being used to facilitate potentially unethical practices within the Spacemesh ecosystem. Given the importance of maintaining fairness and integrity within our network, I felt compelled to report this to the team.

**Reply by pigmej** [reply]
We're aware of these rumors.
